doc/api/cluster.markdown

```diff
@@ -357,18 +357,19 @@ This example will echo back all messages from the master:
       });
     }
 
-### worker.suicide
+### worker.exitedVoluntarily (formerly worker.suicide)
 
 * {Boolean}
 
 Set by calling `.kill()` or `.disconnect()`, until then it is `undefined`.
```

Comment:
That would tie the removal of the worker.suicide alias from the API in some future version and its removal from the docs.

Otherwise, it is unclear what exactly does «formerly worker.suicide» mean.